rascaline/src/pre_calculators/mod.rs

```diff
@@ -0,0 +1,2 @@
+mod bond_atom_triplet_neighbors;
+pub(crate) use bond_atom_triplet_neighbors::{BATripletInfo,BATripletNeighborList};
```

Comment:
Yes, part of this pre-calculator stuff is to separate what stores data in systems rather than in tensormaps.
The other part is to have a clearer dependency tree, given that part of labels/ depends on it.

Reply:
should I still move the precalculator with the calculator, or does the "clearer dependency tree" argument make the new directory worth it?

Reply:
ok, I missed that labels depends on this code. Might make sense to put it in `systems/bond_atoms_neighbors.rs` or something similar then